components/common-go/go-update-wc-deps.sh

```diff
@@ -5,15 +5,16 @@
 
 set -eo pipefail
 
-cd ../..
+DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)
```

Comment:
:+1:

Reply:
Awesome, mine passed too, should be good to go. @akosyakov it would be helpful to get your :heavy_check_mark: (as it'd cover Staff and Experience).